src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -30,9 +30,20 @@
 
 namespace kafka {
 
-bool is_not_supported(const ss::sstring& name) {
-    return name == "min.insync.replicas" || name == "flush.messages"
-           || name == "flush.ms";
+static constexpr std::array<std::string_view, 7> supported_configs{
+  {"compression.type",
+   "cleanup.policy",
+   "message.timestamp.type",
+   "segment.bytes",
+   "compaction.strategy",
+   "retention.bytes",
+   "retention.ms"}};
+
+bool is_supported(const ss::sstring& name) {
+    return std::any_of(
+      supported_configs.begin(),
+      supported_configs.end(),
+      [name](const std::string_view& p) { return name == p; });
```

Comment:
change the signature to 

`bool is_supported(std::string_view name)` 

not not cause a conversion in this line 7 times.

Reply:
@senior7515 `std::string_view` or `std::string_view&` ?

Reply:
std::string_view  no need for reference.

Reply:
a string_view is already a reference type. so no need for `const-ref` so just `std::string_view` should be enough. 

thanks!